hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/listener/NotifyingTopicListener.java

```diff
@@ -90,14 +91,13 @@ public NotifyingTopicListener(DbProperties dbProperties, ListenerProperties list
     }
 
     private Flux<String> listen() {
-        EmitterProcessor<String> emitterProcessor = EmitterProcessor.create();
-        FluxSink<String> sink = emitterProcessor.sink().onDispose(this::unlisten);
-        channel.handler(sink::next);
+        Sinks.Many<String> sink = Sinks.many().multicast().onBackpressureBuffer();
```

Comment:
```suggestion
        Sinks.Many<String> sink = Sinks.many().unicast().onBackpressureBuffer();
```

in the constructor, at the end, it's turned into a shared flux by `share()`, so `multicast()` here is redundant.

Reply:
Agreed

Reply:
Good call